gapic/schema/api.py

```diff
@@ -68,10 +68,10 @@ def build(cls, file_descriptor: descriptor_pb2.FileDescriptorProto,
                 These are needed to look up messages in imported protos.
         """
         return _ProtoBuilder(file_descriptor,
-            file_to_generate=file_to_generate,
-            naming=naming,
-            prior_protos=prior_protos or {},
-        ).proto
+                             file_to_generate=file_to_generate,
```

Comment:
What auto-formatter are you using? Perhaps we should settle on one and reify it in CI.

Reply:
autopep8, part of my python-mode-hook.
Opened issue #142 as a feature task for CI. Do you have strong feelings on autopep8 vs black or something else? My vote is black because there are no choices to make.

Reply:
I am not willing to use black because it generates non-PEP-8 compliant code. autopep8 is fine with me.

Reply:
Most of what I've read about black indicates that its output is PEP-8 compliant. What parts does it violate?